src/app-layout/skeleton/layout.tsx

```diff
@@ -52,27 +50,20 @@ export function SkeletonLayout({
       style={{
         minBlockSize: `calc(100vh - ${placement.insetBlockStart}px - ${placement.insetBlockEnd}px)`,
         [customCssProps.maxContentWidth]: isMaxWidth ? '100%' : maxContentWidth ? `${maxContentWidth}px` : '',
-        [customCssProps.minContentWidth]: `${minContentWidth}px`,
```

Comment:
Fun fact, we never use this property in the actual layout (neither VR nor Classic). The content can resize below min-width. We only use this value for split panel resize limits, which is not happening inside skeleton, therefore removing it here